tests/scripts/test-e2e-cluster.sh

```diff
@@ -42,16 +42,16 @@ rc=0
 set +e
 # runsuite arguments:
 # suiteid test_tags provider provider_keypath provider_url provider_project_id provider provider_deployment_name llm_model ols_image
-run_suite "azure_openai" "not standalone" "azure_openai" "$AZUREOPENAI_PROVIDER_KEY_PATH" "https://ols-test.openai.azure.com/" "" "0301-dep" "gpt-3.5-turbo" "$OLS_IMAGE"
+run_suite "azure_openai" "not standalone" "azure_openai" "$AZUREOPENAI_PROVIDER_KEY_PATH" "https://ols-test.openai.azure.com/" "" "0301-dep" "gpt-3.5-turbo" "$OLS_IMAGE" "0.5"
```

Comment:
we should just default this in the test and if we feel the need to change it, change the default.  this is our baseline, so it doesn't make much sense to me that we'd need to pass it as an arg